src/cli/extensions/helpers.js

```diff
@@ -0,0 +1,281 @@
+import chalk from 'chalk';
+import resolve from 'resolve';
+import { isString } from 'lodash';
+import semver from 'semver';
+
+import { registerHooks, getHooks, setHooks } from '../../hooks';
+import { registerActions, getActions, setActions } from '../../hooks/actions';
+import { merge } from '../../configuration';
+import { warningLabel, feedbackMessage } from '../../helpers/style';
+import ExtensionError from './error';
+
+export function runPostInits(initialState) {
+    return initialState.postInits.reduce(
+        (state, postInit) =>
+            manageRocObject(
```

Comment:
Yeah that might be better but as we know there is some work to do in terms of naming, refactoring e.t.c. The question is if everything should be done in this PR. Since it's not an external facing API I'm fine with letting some stuff be done at later.